### PR TITLE
GRW-2420 - feat(AccordionContent): keep content mounted for SEO purposes

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Keep `AccordionContent`'s content mounted so it can be used for SEO purposes
* Replace Accordion's CSS animations with JS based animations with _framer motion_

https://user-images.githubusercontent.com/19200662/229769745-a4a49b1e-d30b-4f6e-937d-993f822d2e26.mov

## Justify why they are needed

As discussed [here](https://github.com/radix-ui/primitives/discussions/1481) the purpose of `forceMount` is to hand over lifecycle to javascript animations libraries. By using it, we solve the SEO issue but as consequence radix built in animation handler will not properly work. With that said, I've replaced the CSS animations with framer motion ones so we can have both. 

## Jira issue(s): [GRW-2420](https://hedvig.atlassian.net/browse/GRW-2420)

[GRW-2420]: https://hedvig.atlassian.net/browse/GRW-2420?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2104** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2104" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2104?utm_source=stack-comment).

File: apps/store/src/blocks/AccordionBlock.tsx [hunk 1]
```diff
@@ -18,19 +19,30 @@ type Props = SbBaseBlockProps<{
 }>
 
 export const AccordionBlock = ({ blok, nested }: Props) => {
+  const [openedItem, setOpenedItem] = useState<string>()
```
Comment (gustaveen) [inline apps/store/src/blocks/AccordionBlock.tsx:22]: nit: naming, shouldn't be past tense right?
```suggestion
  const [openItem, setOpenItem] = useState<string>()
```

**Review by gustaveen — APPROVED** [review]
Wohoo, nice work! 🙌 Just some spelling fix

File: apps/store/src/blocks/AccordionBlock.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply apps/store/src/blocks/AccordionBlock.tsx:22]: 🤔 I've followed the same "pattern" from [`Perils`](https://github.com/HedvigInsurance/racoon/blob/main/apps/store/src/components/Perils/Perils.tsx#L70).
Maybe I should change there as well.

**Pull request merged**